WordPress/Classes/Models/Plan.swift

```diff
@@ -106,3 +146,74 @@ extension Plan {
+    case PremiumThemes
+    case GoogleAnalytics
+    case StorageSpace(String)
+    case Support(String)
+    
+    var title: String {
+        switch self {
+        case WordPressSite:
+            return NSLocalizedString("WordPress.com Site", comment: "The site that a user gets by signing up at WordPress.com. A feature provided by every plan.")
+        case FullAccess:
+            return NSLocalizedString("Full Access to Web Version", comment: "A WordPress.com feature that every user gets when they select a plan in the mobile app.")
+        case CustomDomain:
+            return NSLocalizedString("A Custom Site Address", comment: "A feature that users gain when purchasing a paid plan.")
+        case NoAds:
+            return NSLocalizedString("No Ads", comment: "A feature that users gain when purchasing a paid plan. No advertising will be displayed on their site.")
+        case CustomFontsAndColors:
+            return NSLocalizedString("Custom Fonts & Colors", comment: "A feature that users gain when purchasing a paid plan. They can set custom fonts and colors on their site.")
+        case CSSEditing:
+            return NSLocalizedString("CSS Editing", comment: "A feature that users gain when purchasing a paid plan. They can edit the CSS of their site.")
+        case VideoPress:
+            return NSLocalizedString("Video Storage & Hosting", comment: "A feature that users gain when purchasing a paid plan. They can upload and embed videos using VideoPress.")
+        case ECommerce:
+            return NSLocalizedString("eCommerce", comment: "A feature that users gain when purchasing a paid plan. They can embed a store in their site.")
+        case PremiumThemes:
+            return NSLocalizedString("Premium Themes", comment: "A feature that users gain when purchasing a paid plan. They have unlimited access to premium themes.")
+        case GoogleAnalytics:
+            return NSLocalizedString("Google Analytics", comment: "A feature that users gain when purchasing a paid plan. ")
+        case StorageSpace:
+            return NSLocalizedString("Storage Space", comment: "A feature that users gain when purchasing a paid plan. They get a certain level of storage space for uploads.")
+        case Support:
+            return NSLocalizedString("Support", comment: "A feature that users gain when purchasing a paid plan. Customer support such as live chat or email.")
+        }
+    }
+    
+    var webOnly: Bool {
+        switch self {
+        case .CustomDomain, .ECommerce, .GoogleAnalytics:
+            return true
+        default:
+            return false
+        }
+    }
+    
+    var description: String? {
+        switch self {
+        case .StorageSpace(let space):
+            return space
+        case .Support(let type):
+            return type
+        default:
+            return nil
+        }
+    }
+}
+
+func ==(lhs: PlanFeature, rhs: PlanFeature) -> Bool {
+    return lhs.title == rhs.title
```

Comment:
Hmm... the reason I added `Equatable` was simply so that I could find a plan's matching feature compared to the list of all features, for use here: https://github.com/wordpress-mobile/WordPress-iOS/blob/feature/plan-details/WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift#L80

This wouldn't work if it also checked on the associated values etc too. I'm happy to restructure the features and do this differently if we can think of a better way though!